docs/ops.html

```diff
@@ -2586,7 +2574,22 @@ <h5 class="anchor-heading"><a id="kafka_streams_node_monitoring" class="anchor-l
         <td>The total number of records that have been emitted downstream from suppression operation nodes.</td>
         <td>kafka.streams:type=stream-processor-node-metrics,thread-id=([-.\w]+),task-id=([-.\w]+),processor-node-id=([-.\w]+)</td>
       </tr>
- </tbody>
+      <tr>
+        <td>record-e2e-latency-avg</td>
```

Comment:
Technically, the record e2e latency metrics are processor node level metrics that are only available on certain types of nodes. Since the bean regex depends on the level (i.e., the metric group) the regex was wrong before.

Reply:
Ah, thanks. Tbh I had forgotten how we ultimately ended up doing these metrics, thanks for catching this